converters/toZigbee.js

```diff
@@ -3660,6 +3661,146 @@ const converters = {
             await sendTuyaDataPointValue(entity, common.TuyaDataPoints.saswellTempCalibration, value);
         },
     },
+    silvercrest_smart_led_string: {
+        key: ['silvercrest_smart_led_string', 'color', 'brightness', 'scene'],
```

Comment:
Key `silvercrest_smart_led_string` can be removed.

Reply:
key: done